### DISCUSSION TITLE
About Prettier and Schema Validation Status

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [ScratchAddons/ScratchAddons#537] [Ideas]
## Prettier Formatting
We use prettier formatting to keep our codebase readable and easy to edit. Every pull request and commit is automatically formatted using prettier. If you're forking our repository, you should turn on Github Actions for automatic formatting. Find out how below:

**Status:** ![Format code using Prettier](https://github.com/ScratchAddons/ScratchAddons/workflows/Format%20code%20using%20Prettier/badge.svg)
## Schema Validation
In our repository, we use addon.json files to tell the extension what to files to inject, where to load them, and all of the settings on the settings page, among other things. Our schema validation makes sure that our addon.json(s) comply with the way they are meant to be.

**Status:** ![Schema validation](https://github.com/ScratchAddons/ScratchAddons/workflows/Schema%20validation/badge.svg)

## Turning on GitHub Actions
1. Go to your fork
2. On the gray navigation bar, next to "Pull Requests," click on Actions.
3. Turn on Actions. It may already be turned on.

**Comment by GrahamSH-LLK** [comment]
This is a possible wiki page that I'm writing. Any thoughts?

**Comment by mxmou** [comment]
> If you're forking our addon

"If you're forking our repository" might make more sense.

**Reply by GrahamSH-LLK** [reply]
Thanks!
